content/zh/news/releases/1.20.x/announcing-1.20/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 发布 Istio 1.20.0
+linktitle: 1.20.0
+subtitle: 大版本更新
+description: Istio 1.20 发布公告。
+publishdate: 2023-11-14
+release: 1.20.0
+aliases:
+- /zh/news/announcing-1.20
+- /zh/news/announcing-1.20.0
+---
+
+我们很高兴地宣布 Istio 1.20 发布。这是 2023 年内最后一个 Istio 版本，
+我们要感谢整个 Istio 社区对 1.20.0 版本发布所作出的帮助。
+我们要感谢此版本的几位发布经理：来自 DaoCloud 的 `Xiaopeng Han`、
+来自 Google 的 `Aryan Gupta` 和来自 Tetrate 的 `Jianpeng He`。
+这些发布经理们要特别感谢测试和发布工作组负责人 Eric Van Norman (IBM) 在整个发布周期中提供的帮助和指导。
+我们还要感谢 Istio 工作组的维护者以及广大 Istio 社区，在发布过程中提供及时反馈、
+审核和社区测试，以及在确保及时发布方面给予的全力支持。
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.20.0 已得到 Kubernetes `1.25` 到 `1.28` 的官方正式支持。
+{{< /tip >}}
+
+## 新特性 {#what-s-new}
```

Comment:
```suggestion
## 新特性 {#whats-new}
```
We can follow the en anchors: https://istio.io/latest/news/releases/1.20.x/announcing-1.20/#whats-new